Documentation/RC1SupportedScenarios.md

```diff
@@ -0,0 +1,24 @@
+WCF in RC1
+===========================
+
+WCF Features
+------------
+
+
+![Image](http://findicons.com/files/icons/573/must_have/16/check.png) -- Supported: Release quality
+
+![Image](http://findicons.com/files/icons/1155/diagona/16/104.png) --  Supported: Not fully tested
+
+![Image](http://findicons.com/files/icons/573/must_have/16/add.png) -- Not Yet Supported: Future release
+
+![Image](http://findicons.com/files/icons/573/must_have/16/delete.png) -- Not Supported: Will not support
+
+
+
+|Scenario |UWP   |Windows |Linux |Mac OS X  |
+|---------|------|--------|------|----------|
```

Comment:
We'll need to start listing what scenarios should appear in the rows.  They need to be easily recognizable by WCF users and extremely terse.  We may also need a new column for "pivots" or "options" such as:

| Scenario  | Option |
| ------------- | ------------- |
| Http  | Security.None  |
|   | Security.Transport|
|   | Text encoding  |
|   | Binary encoding | 
| Https  | Security.None  |
|   | Security.Transport|
|   | Text encoding  |
|   | Binary encoding | 
| Tcp| Security.None  |
|   | Security.Transport|
|   | Text encoding  |
|   | Binary encoding | 
|   | Duplex | 
| WebSockets | etc  |
|   | etc ||